LoopKit/DeviceManager/PumpManager.swift

```diff
@@ -168,6 +168,9 @@ public protocol PumpManager: DeviceManager {
     /// Returns a dose estimator for the current bolus, if one is in progress
     func createBolusProgressReporter(reportingOn dispatchQueue: DispatchQueue) -> DoseProgressReporter?
 
+    /// Returns the estimated time for the bolus amount to be delivered
+    func estimatedDuration(toDeliver bolusUnits: Double) -> TimeInterval
```

Comment:
Should "bolus" be part of the calling signature somehow?  Might be problematic if we want to define a function to estimate other types of delivery.

Reply:
Agreed! Perhaps `func estimatedDuration(toBolus units: Double) -> TimeInterval`?